src/lib/utils.ts

```diff
@@ -1,13 +1,16 @@
-export function getPreferredTheme() {
+import {Theme} from '../hooks/useTheme';
```

Comment:
Little bit odd importing from hooks in the /lib directory. Maybe better to make a new file `/lib/theme.ts` and move `getPreferredTheme` function into there along with that `Theme` type.